From 56b01182f04b0201e34c3d05a54669367fb9005f Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Wed, 28 Aug 2024 14:08:39 +0200
Subject: [PATCH] u-boot: Disable mkeficapsule

It depends on gnutls, which we can not rely on being present and we
don't need it anyway.
---
 include/u-boot.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/u-boot.mk b/include/u-boot.mk
index 8d946d1122..ce0d69ac56 100644
--- a/include/u-boot.mk
+++ b/include/u-boot.mk
@@ -118,6 +118,7 @@ define Build/Configure/U-Boot
 		$(UBOOT_CONFIGURE_VARS) \
 		$(firstword $(UBOOT_CONFIG))_config \
 		$(wordlist 2,$(words $(UBOOT_CONFIG)),$(UBOOT_CONFIG:%=%.config))
+	sed -i 's|^CONFIG_TOOLS_MKEFICAPSULE=*|# CONFIG_TOOLS_MKEFICAPSULE is not set|' $(PKG_BUILD_DIR)/.config
 	$(if $(strip $(UBOOT_CUSTOMIZE_CONFIG)),
 		$(PKG_BUILD_DIR)/scripts/config --file $(PKG_BUILD_DIR)/.config $(UBOOT_CUSTOMIZE_CONFIG)
 		+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) CROSS_COMPILE=$(TARGET_CROSS) $(UBOOT_CONFIGURE_VARS) oldconfig)
-- 
2.52.0

